mypy_boto3_builder/stubs_static/types-aioboto3/s3/cse.pyi

```diff
@@ -99,8 +99,8 @@ class S3CSE:
     async def __aenter__(self: _R) -> _R: ...
     async def __aexit__(
         self,
-        exc_type: Optional[Type[Exception]],
-        exc_val: Optional[Exception],
+        exc_type: Optional[Type[BaseException]],
+        exc_val: Optional[BaseException],
```

Comment:
Should be moved to https://github.com/youtype/types-aioboto3/blob/main/aioboto3-stubs/s3/cse.pyi